python-sdk/examples/breast_cancer_detection/breast_cancer_detection.py

```diff
@@ -0,0 +1,178 @@
+"""
+This is a reproduction of pachyderm's breast cancer detection example in 
+python:
+  https://github.com/pachyderm/examples/tree/master/breast-cancer-detection
+
+This example is intended to be run from the 
+pachyderm/python-sdk/examples/breast_cancer_detection directory.
+"""
+import os
+
+from pachyderm_sdk import Client
+from pachyderm_sdk.api import pfs, pps
+
+
+def main(client: Client):
+    # Create repos for models and sample data
+    models = pfs.Repo.from_uri("models")
+    client.pfs.create_repo(repo=models)
+
+    sample_data = pfs.Repo.from_uri("sample_data")
+    client.pfs.create_repo(repo=sample_data)
+
+    # Populate the repos with data and commit
+    models_src_path = os.getcwd() + "/models"
```

Comment:
This is a bit pedantic, but the safer thing to do here is to do the following:
```python
from pathlib import Path
models_src_path = Path(__file__).parent / "models"
```
The above uses the location of this script file as the reference point rather than the cwd. Additionally since you use the cwd call in a couple places you could have a `SCRIPT_DIR = Path(__file__).parent`